client/script.js

```diff
@@ -0,0 +1,65 @@
+
+const searchButton = document.querySelector('.submit')
+const form = document.querySelector('#search-form')
+
+  const  init= async (e) => {
+   
+        e.preventDefault();
+        
+        clearArtistList();
+        
+        const randomInput = e.target.text.value;
+
+        if(randomInput.toLowerCase() === "random")
+        {
+            getRandom();
+        }
+        else
+        {
+            try
+            {
+
+                const response = await fetch('http://localhost:3000/artists')
+    
+                const data = await response.json()
+    
+                setData(data)
+     
+            } catch (err) {
+                console.log(err)
+            }
+        }
+    }
+  
+function getRandom(){
```

Comment:
Consistency around your syntax for this particular file would be awesome - the ones above are all spaced out and the curly brackets on the functions below are all in different places. But this is me being picky. Your code however, is spectacular 👍